app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditFragmentTest.kt

```diff
@@ -158,6 +158,25 @@ class ProfileEditFragmentTest {
     }
   }
 
+  @Test
+  fun testProfileEdit_startWithUserProfile_clickProfileDeletionButton_deleteSnackbarIsVisible() {
+    launchFragmentTestActivity(internalProfileId = 1).use {
+      onView(withId(R.id.profile_delete_button)).perform(click())
+      onView(withId(android.R.id.button1)).perform(click())
+      onView(withId(com.google.android.material.R.id.snackbar_text))
+        .check(matches(withText(R.string.profile_edit_delete_success)))
+    }
```

Comment:
Could you try this matcher?
```suggestion
     launchFragmentTestActivity(internalProfileId = 1).use {
      onView(withId(R.id.profile_delete_button)).perform(click())
      testCoroutineDispatchers.runCurrent()
      onView(withText(R.string.profile_edit_delete))
        .inRoot(isDialog())
        .perform(click())
      testCoroutineDispatchers.runCurrent()
      onView(allOf(withText(R.string.profile_edit_delete_success)))
        .check(matches(isDisplayed()))
    }
```